apps/widget-configurator/src/app/configurator/index.tsx

```diff
@@ -257,7 +257,7 @@ export function Configurator({ title }: { title: string }) {
       </Drawer>
 
       <Box sx={ContentStyled}>
-        {params && provider && (
+        {params && (
```

Comment:
This part is also in my open PR https://github.com/cowprotocol/cowswap/pull/3846/files#diff-0aaf944b2edf229ebee7167a45771a40db79ec1b37f971598161a5d2976077dfL260